crates/astria-conductor/src/executor/client.rs

```diff
@@ -68,7 +71,14 @@ impl Client {
         transactions: Vec<Vec<u8>>,
         timestamp: Timestamp,
     ) -> eyre::Result<Block> {
-        let transactions = transactions.into_iter().map(Bytes::from).collect();
+        use prost::Message;
+
+        let transactions = transactions
+            .into_iter()
+            .map(|tx| RollupData::decode(tx.as_slice()))
+            .collect::<Result<_, _>>()
+            .wrap_err("failed to decode tx bytes as RollupData")?;
```

Comment:
just calling out that this will eventually halt conductor

Reply:
Yep this is expected, if we have data that can't be deserialized it's an issue that we shouldnt skip over and just execute.